docs/content/docs/deployment/cli.md

```diff
@@ -78,6 +78,11 @@ There is another action called `run-application` available to run the job in
 [Application Mode]({{< ref "docs/deployment/overview" >}}#application-mode). This documentation does not address
 this action individually as it works similarly to the `run` action in terms of the CLI frontend.
 
+The `run` and `run-application` commands support passing additional configuration parameters via the
+`-D` argument. For example setting the [maximum parallelism]({{< ref "docs/deployment/config#pipeline-max-parallelism" >}}#application-mode) for a job can be done by setting `-Dpipeline.max-parallelism=120`. This argument is very useful for configuring per-job or application mode clusters, because you can pass any configuration parameter to the cluster, without changing the configuration file.
```

Comment:
Any restrictions on line length? This one is very long (in contrast to the previous one)

Reply:
Except for a few years of using Eclipse, I've never managed in my life to set up automated line wrapping. Main reason is that I'm not really convinced of that concept.
I'll do some wrapping to my change.